org.civicrm.reqangular/src/common/angular-date/src/controllers/DatePickerController.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @extends DatepickerController
+ */
+function DatePickerController($scope, $controller, DateFactory, $log){
+    $scope.implements = 'DatepickerController';
+
+    var me = this,
+        ngModelCtrl = { $setViewValue: angular.noop };
+
+    angular.extend(this, $controller('DatepickerController', {
+        $scope: $scope,
+        $attrs: {}
+    }));
+
+
+    $scope.isActive = function(dateObject) {
+        if (me.compare(dateObject.date, me.activeDate) === 0) {
+            $scope.activeDateId = dateObject.uid;
+            return true;
+        }
+        return false;
+    };
+
+    me.parseDate = function(date){
+        var formatted = DateFactory.createDate(date).format('DD/MM/YYYY');
+
+        var newDate = DateFactory.moment(formatted, 'DD/MM/YYYY');
+        return newDate.toDate();
+    };
+
+    /* Overriding Methods */
+    me.render = function() {
+        if ( ngModelCtrl.$modelValue ) {
+
+            var date = me.parseDate(ngModelCtrl.$modelValue),
+                isValid = !isNaN(date);
```

Comment:
Need consistency on how new `var`s are declared + defined, either
```js
var foo = 'foo';
var bar = 'bar';
```
or
```js
var foo = 'foo', bar = 'bar';
```
personally i'm more inclined for the former (i use the latter only when i declare them), but anyway the syntax must be consistent